### PR TITLE
Custom domains: Add note about redirect URIs

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description
- **What's changed?** Adds note about redirect URIs in SAML app settings, IdP settings, and Authorization Server settings.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-629975](https://oktainc.atlassian.net/browse/OKTA-629975)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 1]
```diff
@@ -52,6 +52,18 @@ You can create up to three custom domains with multibrand customizations and up
 
 You can only preview or visit a branded page (such as viewing brand assets applied to the Okta-hosted sign-in page) after you map to a custom domain. After you create a brand, map it to a custom domain. Then you can make further customizations, preview them, and publish them.
 
+#### Redirect URIs
+
+Multibrand orgs use dynamic issuer mode for IdP. As a result, Okta uses the domain from the authorize request as the domain for the redirect URI when returning the authentication response. The Admin Console UI displays the org's Okta subdomain when the org has multiple custom domains configured.
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:57]: I understood the acronym "IdP" as "Identity Provider", which doesn't really work in this sentence. Is it correct? (Applies to previous section as well)

**Review by brianduffield-okta — APPROVED** [review]
Some minor nits and one question.

File: packages/@okta/vuepress-site/docs/concepts/brands/index.md [hunk 2]
```diff
@@ -86,10 +86,18 @@ If you want to use the Admin Console to send a branded email, consider the follo
...
-### Multibrand and authorization servers
+### Multibrand and redirect URIs
 
 Multibrand orgs use dynamic issuer mode for IdP. As a result, Okta uses the domain from the authorize request as the domain for the redirect URI when returning the authentication response. The Admin Console UI displays the org's Okta subdomain when the org has multiple custom domains configured.
 
+URIs that you use in the following settings revert to the Okta subdomain:
+
+- [SAML apps](https://help.okta.com/okta_help.htm?id=ext-apps-about-saml)
+- [IdP settings - Redirect URI](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-oidc)
+- [Authorization Server settings](/docs/guides/customize-authz-server/main/)
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/concepts/brands/index.md]: Thank you. Updated.

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:57]: I'll dig and get more info. Thanks!

**Pull request merged**